### PR TITLE
Upgrade tailwindcss from v0.6  to v1.1.2

### LINKED ISSUE #109: ✨ Upgrade to Tailwind CSS v1
## Description
Currently, this project is using Tailwind v0.x but I'd like it to be upgraded to use Tailwind v1. Here's [an upgrade guide](https://tailwindcss.com/docs/upgrading-to-v1/) which might be helpful.

### PR DESCRIPTION
**Pull request by SARAN-thala**
# Description
- Upgrade tailwindcss to major version
- Minor version upgrade on laravel-mix-purgecss plugin

# Issues
closes #109

### CODE REVIEW

**Comment by damcclean** [comment]
Hey @SARAN-thala, thanks for your contribution!

Unfortunately, you made your pull request at the same time I was tidying up the styles of this project so there are a few things that are conflicting.

Feel free to restart this pull request or rebase so you get the changes.

**Comment by SARAN-thala** [comment]
Sure will do that

File: assets/css/app.css [hunk 1]
```diff
@@ -2,7 +2,18 @@
...
+@tailwind base;
+
+ul {
+  list-style-type: disc;
+  padding-left: theme('padding.4');
+}
+
+ol {
+  list-style-type: decimal;
+  padding-left: theme('padding.4');
```
Comment (damcclean) [inline assets/css/app.css]: hey, please also change the `config()` method to being `theme()`

File: tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+    userSelect: ['responsive'],
+    verticalAlign: ['responsive'],
+    visibility: ['responsive'],
+    whitespace: ['responsive'],
+    width: ['responsive'],
+    wordBreak: ['responsive'],
+    zIndex: ['responsive'],
+  },
+  corePlugins: {
+    container: false
```
Comment (damcclean) [inline tailwind.config.js]: hey, the container isn't working and this is likely why... 🤔

Comment (SARAN-thala) [reply tailwind.config.js]: remove the container value

**Comment by damcclean** [comment]
hey, remember to change `config` to `theme` in the stylesheet file.

Also, would you mind fixing some of the font weights as they seem to all be really thin now.

**Comment by SARAN-thala** [comment]
Got it. going to change config to theme

File: assets/css/app.css (hunk 1 above)
Comment (SARAN-thala) [reply assets/css/app.css]: fixed it

**Comment by SARAN-thala** [comment]
fixing mono font aswell

**Comment by damcclean** [comment]
> fixing mono font aswell

yeah, we had a custom font setup in the old tailwind config called `space`.

**Comment by SARAN-thala** [comment]
fixed it now

**Comment by damcclean** [comment]
Perfect! Thanks for all your help on this pull request.

**Pull request merged**